.ci/wercker.sh

```diff
@@ -136,11 +136,8 @@ no-error-htmlunit)
   CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
                      --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
   echo CS_version: ${CS_POM_VERSION}
-  echo "checkouting project sources ..."
-  svn -q export https://svn.code.sf.net/p/htmlunit/code/trunk/htmlunit@r14923 htmlunit
-  cd htmlunit
- sed -i "s/            <version>2.28-SNAPSHOT/            <version>2.28-20171106.080245-12/" pom.xml
-  echo "Running checkstyle validation ..."
+  checkout_from https://github.com/HtmlUnit/htmlunit
+  cd .ci-temp/htmlunit
   mvn -e compile checkstyle:check -Dcheckstyle.version=${CS_POM_VERSION}
   cd ../
```

Comment:
cd changed from `cd htmlunit` to `cd .ci-temp/htmlunit`. Since we are `cd`ing into a second directory, don't we need to `cd` out an additional directory too?

Reply:
Actually, we are deleting the `htmlunit` folder so we should only go out 1. Please double confirm.

Reply:
What I have is correct, there are other errors in this file, like what you've mentioned.  There are a few that do that, but they end up living in .ci-temp and never getting deleted, just updated, which I guess could be either good or bad, but the intended action of rm'ing the directories isn't carried out.

Reply:
![image](https://user-images.githubusercontent.com/31252532/76466692-85538200-63be-11ea-9485-0ee0761a6d76.png)

Reply:
Would you like me to tidy this file up along with the hipparchus directory removal and submit as one PR?

Reply:
Yes, feel free to make another PR ensure the temp folder is tidied up. I was asking @romani since he knows linux better than me: can we remove the f option for rm -rf but not make it prompt? This way it will complain if directory doesn't exist and catch this problem for us.
http://man7.org/linux/man-pages/man1/rm.1.html

Reply:
It should only prompt you to rm if you use the -i flag.

Reply:
In my experience (I have much more with Linux than with programming), only the -i flag will make it prompt when rm'ing, despite what the man page says.  Since it is a git repo though, it might be necessary to use -f because of the protected files.  I'll take a look tonight.

Reply:
Let's move all noticed points to improve to separate issue/PRs .

It is better to merge this PR sooner.

Reply:
@nmancus1 Do you still plan to do this?

Reply:
@rnveach yes, I completed it today. I just wanted to run through and test each one again before submitting PR.